libs/vue/src/builders/browser/schema.json

```diff
@@ -96,6 +96,11 @@
         }
       },
       "additionalProperties": false
+    },
+    "vueNxConfigPath": {
+      "type": "string",
+      "description": "The path to the vue-nx.config.js file, relative to the current workspace.",
+      "default": "vue-nx.config.js"
```

Comment:
Remove default value. This should work similarly to `main`, where the full path must be provided to the file.

Reply:
Like it says in the description, you can provide a path to any file. In this case it assumes a default file name. If this file exists it will use it, otherwise it will ignore it. Not sure what would be wrong using a default file naming (something like `configure-webpack.js`)?

Reply:
Let's not make the file location configurable. Vue CLI assumes `vue.config.js` is at the root of the project, so let's assume `configure-webpack.js` is at the root of the project. Plus, if `vue.config.js` support is added in the future this assumption will help ease the migration.

Reply:
I kept it configurable as it does not really matter implementation wise.

Reply:
We don't want users to be able to put the file where ever they'd like though. We want `configure-webpack.js` to work similarly to [`vue.config.js`](https://cli.vuejs.org/config/#vue-config-js). Please remove the option.